### DISCUSSION TITLE
Intermittent issues with downloading/extracting actions in hosted runners

### DISCUSSION THREAD

**Discussion by strobeflash** [community/community#17656] [Actions] [answered]
The action actions/setup-python@v2 fails frequently, someone else confirmed current issues with downloading/extracting actions on irc.

Runner logs:
https://github.com/strobeflash/ics-py/runs/6704029660?check_suite_focus=true

Local debugging (manual download and unpacking of tar archive)
https://pastebin.com/8Ue8Kwyx

What could be the reason?

**Comment by tripleee** [comment, marked as answer]
Apparently related: https://github.com/actions/checkout/issues/815

**Comment by strobeflash** [comment]
Works for me now, can be closed.
